### PR TITLE
use % formatter

### PR DESCRIPTION
**Pull request by EdwardBetts**

### CODE REVIEW

**Comment by pllim** [comment]
I don't understand what this is supposed to fix.

File: astropy/utils/console.py [hunk 1]
```diff
@@ -658,9 +658,9 @@ def _update_ipython_widget(self, value=None):
             self._widget.value = 0
 
         # Calculate percent completion, and update progress bar
-        percent = (value/self._total) * 100
-        self._widget.value = percent
-        self._widget.description =' ({0:>6s}%)'.format('{0:.2f}'.format(percent))
+        percent = (value/self._total)
```
Comment (bsipocz) [inline astropy/utils/console.py]: I would either keep the multiplication here, as this is not the percent but the fractional value in this form or rename the variable to ``frac`` as above.

**Comment by astrofrog** [comment]
@EdwardBetts - can you add a description of what this does and fixes?

File: astropy/utils/console.py [hunk 2]
```diff
@@ -658,9 +658,9 @@ def _update_ipython_widget(self, value=None):
...
+        self._widget.value = percent * 100
+        self._widget.description =' ({:>6.2%})'.format(percent))
```
Comment (pllim) [inline astropy/utils/console.py]: Mismatched parentheses (syntax error).

**Comment by EdwardBetts** [comment]
There's no need to call format twice in the same line. You're trying to format a number then add some spaces. You can specify the width when formatting the number.

I'm also using the Python '%' formatter, this multiplies the parameter by 100 and adds a % symbol.

**Comment by saimn** [comment]
Using this % type specifier is indeed a good simplification, I didn't know about this one :
(https://docs.python.org/2/library/string.html#grammar-token-type)

**Comment by pllim** [comment]
Thanks for the clarifications, @EdwardBetts ! I restarted the one failed build that seems unrelated to this change.

**Comment by pllim** [comment]
Since this is more of a formatting nit-picking change with no actual change to code behavior, I think we don't need a change log here and can save this for 1.2.

**Comment by pllim** [comment]
All tests passed. Anyone else want to comment on this? It does not look too controversial, so I am okay with merging this soon.

**Comment by astrofrog** [comment]
I learned something today! (the ``%`` formatter) - :+1: from me

@EdwardBetts - thanks for the PR! Just in terms of procedure, in future please always provide a description for a PR even if the changes seem obvious :)

**Comment by EdwardBetts** [comment]
@astrofrog - thanks, will do.

**Pull request merged**